modernpython/langPack.py

```diff
@@ -49,30 +126,127 @@ def _set_default(text, render):
 def _set_type(text, render):
     return _get_type_and_default(text, render)[0]
 
+def _lower_case_first_char(str):
+    return str[:1].lower() + str[1:] if str else ""
+
+# attributes should never have the same name as a class the may map to
+# Python won't be happy with that, so let's normalize attribute names
+# and leverage aliasing in pydantic to use the cim attribute capitalization
+# during value setting.
```

Comment:
I am pretty sure this is a pydantic bug: https://github.com/pydantic/pydantic/issues/8240
Aliases would create more issues than it would solve I think.

Reply:
Hum, not something that can be fixed easily... But I really do not like the `alias` way. I'll keep thinking about it.

Reply:
yeah, it seems that some python changes are needed: https://github.com/pydantic/pydantic/issues/7327#issuecomment-1829762952

Reply:
if we make work 1 resource 1 file with cyclic references, we could use aliases for the imported class names, and this may do.

Reply:
> yeah, it seems that some python changes are needed: [pydantic/pydantic#7327 (comment)](https://github.com/pydantic/pydantic/issues/7327#issuecomment-1829762952)

It might end up being fixed! https://github.com/pydantic/pydantic/pull/8243#issuecomment-1832702035 Fingers crossed, here.